app/controllers/api/service_orders_controller.rb

```diff
@@ -61,7 +61,7 @@ def add_request_to_cart(workflow)
     end
 
     def create_service_order(data)
-      ServiceOrder.new(data).tap do |service_order|
+      ServiceOrderCart.new(data).tap do |service_order|
```

Comment:
We can't do this, what if this is used by v2v?

Reply:
@fdupont-redhat Does V2V use this endpoint?

Reply:
This method is only called when `service_requests` is blank above.  See https://github.com/ManageIQ/manageiq-api/blob/master/app/controllers/api/service_orders_controller.rb#L10-L11

The only caller for this is the SUI when it wants to create an empty cart.  Or someone calling the API directly for cart operations, it is not used by v2v.

v2v and OpsUI service ordering will create a new request and the ServiceOrder is a by-product of that operation.  So it is safe to assume that this method always wants to create a `ServiceOrderCart` object.

Reply:
Cool, thanks for the clarification @gmcculloug